StarknetCongo/solo-safe-landing-page
# Add ESLint and Stylelint

### Added ESLint and Stylelint for Code Consistency

#### Issue: https://github.com/StarknetCongo/solo-safe-landing-page/issues/1

#### Changes Made:
- Integrated ESLint for JavaScript and JSX files to enforce coding standards and identify errors.
- Integrated Stylelint for CSS and SCSS files to ensure consistent styling across the project.
- Configured ESLint and Stylelint to use standard rules for React projects.
- Fixed existing ESLint and Stylelint errors in the codebase to enhance code quality.

#### How to Test:
- Run `npx eslint "**/*.{js,jsx}"` to check for ESLint errors.
- Run `npx stylelint "**/*.{css,scss}"` to check for StyleLint errors.
- Use `npx eslint "**/*.{js,jsx}" --fix` and  `npx stylelint "**/*.{css,scss}" --fix` to automatically fix ESLint and Stylelint errors where possible.

#### Additional Notes:
- ESLint and Stylelint configurations are located in `.eslintrc.json` and `.stylelintrc.json`, respectively.
- Ensure all new code adheres to ESLint and Stylelint rules before merging.

This PR sets up essential linting tools and improves code quality by fixing existing issues. It helps maintain consistency and readability throughout the project.
